tools/ExecuteCheckOutputFilesAndClean.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
I think my suggestion to put this file in `tools/` was a mistake, sorry. Probably `cmake/` or a subdirectory therein would be a better place because the script only works once the cmake variables are filled in.